### ISSUE TITLE
Default logging configuration

### ISSUE THREAD

**Issue by eblot** [TheCellule/python-bleson#82]
It would be nice if logger's basicConfig would not be called by default, at least not from the global scope, as the logging system gets a default configuration whenever Bleson is imported, which add work to revert this config and use a dedicated config.

**Comment by eblot** [comment]
For now I'm using
````py
Logger.root.addHandler(NullHandler())
````
before importing Bleson, so that basicConfig does nothing.
There may be a better way.

**Comment by WayneKeenan** [comment]
yeah, it's a bit naughty that a library messes with global logging, I'll look at changing it.
